src/gui/mod.rs

```diff
@@ -369,7 +369,11 @@ impl App {
                                     egui::Button::new(job).small().stroke(egui::Stroke::NONE)
                                 };
 
-                                let res = ui.add_enabled(false, button);
+                                let res = if hover_str.is_empty() {
```

Comment:
To indicate the presence or absence of the hover value, `Option<&str>` should be used:

```rust
if let Some(hover_str) = hover_str {
    // display hover_str
} else {
    // no hover string
}
```